akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FormFieldDirectivesSpec.scala

```diff
@@ -87,7 +87,7 @@ class FormFieldDirectivesSpec extends RoutingSpec {
       }
     "work even if only a FromStringUnmarshaller is available for a multipart field with custom Content-Type" in {
       Post("/", multipartFormWithTextHtml) ~> {
-        formFields(('firstName, "age", 'super ? false)) { (firstName, age, vip) =>
+        formFields(("firstName", "age", "super".withDefault(false))) { (firstName, age, vip) =>
```

Comment:
```suggestion
        formFields(("firstName", "age", "super".withDefault(false))) { (firstName, age, vip) =>
```

(not really a suggestion, but a comment)

I find this syntax (no Symbol + name-based method) a lot cleaner. 👍🏼